Fix styles for right sidebar in now playing view

kittaro opened the pull request:
This fix for Issue #49

about:
- update css to use current class names (.xjf0Pj3YnoegOkJUpaPS, .EnViFhuIR5WVeEopJHu3, .wfJD_yK4h7xnpTmrh62U) and use root patch instead catching
- add dimming on top bar with brightness(0.6) (this looks cooler + icons and text are now more visible on very light background of album cover)

upd: fix 1px shadow offset by reducing size of _only background underlayer_, not entire element container (unrelated to `border-bottom: 1px](border-bottom: 1px solid rgba(var(--spice-rgb-selected-row),0.2);` - something else are broken):

before ↓
<img width="362" alt="before" src="https://github.com/user-attachments/assets/8e26c35b-3a4f-4d44-995d-dec4f2069bbb" />
<img width="362" alt="after" src="https://github.com/user-attachments/assets/f7dd4fae-3ecf-4fc0-a794-cc35c45964ff" />
after ↑

kittaro committed:
Refactor styles for right sidebar in now playing view

- update the CSS to use current Spotify UI class names (.xjf0Pj3YnoegOkJUpaPS, .EnViFhuIR5WVeEopJHu3, .wfJD_yK4h7xnpTmrh62U)
- refactoring
- add dimming on top bar with brightness(0.6) (this looks cooler + icons and text are now more visible on very light background of album cover)

kittaro committed:
small fix for the bottom shadow offset

i was with 1px weird indent, very annoying

ohitstom commented:
can you make sure this works on the old version to, for example merge the two selectors instead of overwriting with a new one, then install the recommended versions in the readme and verify with screenshots before i can merge please! :)

as for your other fixes / changes,  they look and sound reasonable but i havent got an environment to test them in so could you also test these design choice changes on the recommended versions before i can merge ❤️

kittaro committed:
compatibility: spotify=1.2.51; spicetify v2.38.5

add selectors from recommended version in repo

kittaro commented:
so, added support for recommended version in repo. all fixes (brightness + 1px offset) are also applied for this version as well
old classes were also rewritten like in new one, and everything seems to work fine (pls ignore win7 vibe, it was the only vm installed):

<img height="997" alt="main" src="https://github.com/user-attachments/assets/2a94b468-9b03-4a36-b864-54b754248aed" />

<img height="601" alt="version info" src="https://github.com/user-attachments/assets/49ea258b-0681-4c72-8ba0-3d30b79b75e6" />
